src/apis/handlers.rs

```diff
@@ -86,7 +86,7 @@ pub async fn remove_torrent_from_whitelist_handler(
 }
 
 pub async fn reload_whitelist_handler(State(tracker): State<Arc<Tracker>>) -> Response {
-    match tracker.load_whitelist_from_database().await {
+    match tracker.load_whitelist().await {
```

Comment:
Hi @WarmBeer I renamed the `load_whitelist` to `load_whitelist_from_database` because some of them had the suffix `from_database` and others don't:

- `load_keys_from_database`
- `load_whitelist_from_database`
- `load_torrents_from_database`